controllers/toolchaincluster/toolchaincluster_controller.go

```diff
@@ -16,13 +16,14 @@ import (
 )
 
 // NewReconciler returns a new Reconciler
-func NewReconciler(mgr manager.Manager, namespace string, timeout time.Duration) *Reconciler {
+func NewReconciler(mgr manager.Manager, namespace string, timeout time.Duration, useClusterRL bool) *Reconciler {
 	cacheLog := log.Log.WithName("toolchaincluster_cache")
 	clusterCacheService := cluster.NewToolchainClusterService(mgr.GetClient(), cacheLog, namespace, timeout)
 	return &Reconciler{
 		client:              mgr.GetClient(),
 		scheme:              mgr.GetScheme(),
 		clusterCacheService: clusterCacheService,
+		useClusterRL:        useClusterRL,
```

Comment:
I wasn't sure what the variable purpose was at first, maybe we could rename it to `useClusterRole` or `setClusterRole`. 

Maybe we could make this a string and set it to `cluster.Tenant` in host-operator and leave it empty in member operator. Thus check if the variable is not empty and only then set the role in the toolchaincluster cr.

WDYT?

Reply:
let's drop this field, we don't need it - see my other comments.